databases/backends/postgres.py

```diff
@@ -263,7 +267,7 @@ def _compile(self, query: ClauseElement) -> typing.Tuple[str, list, tuple]:
 
     @staticmethod
     def _create_column_maps(
-        result_columns: tuple,
+            result_columns: tuple,
```

Comment:
I think it's mixing tab and spaces in multiple files. You can use `./scripts/lint` to fix that.